contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.1.0--3.2.0.sql

```diff
@@ -6,11 +6,12 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 DROP CAST IF EXISTS(NUMERIC AS sys.BIT);
 CREATE CAST (NUMERIC AS sys.BIT) WITH FUNCTION sys.numeric_bit (NUMERIC) AS IMPLICIT;
 
-CREATE FUNCTION sys.int4varbinarydiv(leftarg int4 , rightarg sys.bbf_varbinary)
+CREATE OR REPLACE FUNCTION sys.int4varbinarydiv(leftarg int4 , rightarg sys.bbf_varbinary)
 RETURNS int4
 AS 'babelfishpg_common', 'int4varbinary_div'
 LANGUAGE C IMMUTABLE STRICT PARALLEL SAFE;
 
+DROP OPERATOR IF EXISTS sys./ (int4, sys.bbf_varbinary);
```

Comment:
we should not drop operator in this way. If some objects (views) are depended on this operator then upgrade might get broken.

Reply:
+1, please take a look [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/commit/9425602dc77d1cf3d9ca8e08ee99a34de1ee5296#diff-e42e328779ca480befb9bcd704611cd70ac49cc2d59cc0f794003138f3fe66baR939), we can use a `DO` block in a way that we only create this operator if it does not exists instead of dropping it first which is quite dangerous during upgrade.

Reply:
This would be the best solution. @hash-16 Can you please try out this?

Reply:
Addressed